apps/legacyDash/src/app.ts

```diff
@@ -12,6 +12,7 @@ const logger = scopedLogger("app");
 export class App extends DockPanel {
     private _url = new Comms.ESPUrl().url(document.URL);
     private _ddlv1 = new JSONEditor();
+    private _ddlv2XXX = new JSONEditor();
```

Comment:
What is this compared to `_ddlv2`? Maybe `_ddlv2XXX` should be named more appropriately?